omero/sysadmins/unix/server-centos6.txt

```diff
@@ -97,27 +82,175 @@ Install Java::
+earlier in `settings.env`, so you may need to copy and/or source this file as
+the omero user.
+
+.. literalinclude:: walkthrough/step04_all_omero.sh
+
+
+A convenient way to do it is to download :download:`step04_all_omero.sh <walkthrough/step04_all_omero.sh>`
+and place it with `settings.env` (already edited in a previous step) into the
+``/home/omero`` directory, and run the following command as omero::
+
+    su - omero -c "bash -eux step04_all_omero.sh"
+
+Installing Nginx
+----------------
+
+See also :doc:`install-web/install-nginx`.
+
+To install Nginx, as **root**, run::
+	
+	yum install nginx
+
+In the virtual environment, install the requirements to run OMERO.web using the requirements file::
+
+	/home/omero/omeroenv/bin/pip2.7 install -r ~omero/OMERO.server/share/web/requirements-py27-nginx.txt
+
+Deactivate the virtual environment::
+
+    deactivate
+
+Copy the Nginx OMERO configuration file into the Nginx configuration directory, and disable the default configuration and start Nginx::
+
+    mv /etc/nginx/conf.d/default.conf /etc/nginx/conf.d/default.disabled
+    cp ~omero/OMERO.server/nginx.conf.tmp /etc/nginx/conf.d/omero-web.conf
+    service nginx start
+
+Installing Apache
+-----------------
+
+See also :doc:`install-web/install-apache`.
+
+The version of Apache installed is 2.2, see the note below if you wish to use
+version 2.4.
+
+To install Apache 2.2, as **root**, run::
+	
+    yum install httpd
+    # install mod_wsgi compiled against Python 2.7
+    yum install python27-mod_wsgi
+
+In the virtual environment, install the requirements to run OMERO.web using the requirements file::
+
+    /home/omero/omeroenv/bin/pip2.7 install -r ~omero/OMERO.server/share/web/requirements-py27-apache.txt
+
+As the **omero** system user, configure OMERO.web:
+
+.. literalinclude:: walkthrough/setup_omero_apache22.sh
+
+A convenient way to do it is to download :download:`setup_omero_apache22.sh <walkthrough/setup_omero_apache22.sh>` in the ``/home/omero`` directory and run as omero::
+
+    su - omero -c "bash -eux setup_omero_apache22.sh"
+
+Deactivate the virtual environment::
+
+    deactivate
+
+Open the Apache OMERO configuration file i.e. :file:`omero/OMERO.server/apache.conf.tmp`, and add the line below to the ``python-path`` parameter of ``WSGIDaemonProcess``::
+
+	/home/omero/omeroenv/lib64/python2.7/site-packages
+
+
+Copy the Apache OMERO configuration file into the Apache configuration directory
+and start Apache::
+
+	cp ~omero/OMERO.server/apache.conf.tmp /etc/httpd/conf.d/omero-web.conf
+
+	chkconfig httpd on
+	service httpd start
+
+
+.. note:: If you wish to use Apache 2.4 with mod_wsgi, see :download:`docker_virtualenv_ius_apache24.sh <walkthrough/docker_virtualenv_ius_apache24.sh>` for a possible solution.
+
+
+Final tasks
+-----------
+
+The installation should now be complete.
+To run OMERO, secure OMERO and perform regular tasks, please read the
+relevant sections of :doc:`server-linux-walkthrough`.
```

Comment:
Better to do in another PR since I have a PR modify the linux-walkthrough https://github.com/openmicroscopy/ome-documentation/pull/1351